### PR TITLE
[dv/sram] initial fcov plan

### PR DESCRIPTION
**Pull request by udinator**
this PR adds the initial functional coverage plan for SRAM_CTRL.

Signed-off-by: Udi Jonnalagadda <udij@google.com>

### CODE REVIEW

File: hw/ip/sram_ctrl/data/sram_ctrl_base_testplan.hjson [hunk 1]
```diff
@@ -147,4 +147,40 @@
...
+      name: raw_hazard_cg
+      desc: '''
+            Covers RAW hazard occurrences in the SRAM, and also covers the occurrences that require
+            data forwarding (e.g. colliding accesses to same memory line).
+            '''
+    }
+    {
+      name: b2b_access_types_cg
+      desc: '''
+            Covers different of transaction types (read/write) that can create a b2b scenario.
```
Comment (cindychip) [inline hw/ip/sram_ctrl/data/sram_ctrl_base_testplan.hjson:176]: Covers different transaction types. Do we need the `of` here?

**Review by cindychip — APPROVED** [review]

File: hw/ip/sram_ctrl/data/sram_ctrl_base_testplan.hjson (hunk 1 above)
Comment (udinator) [reply hw/ip/sram_ctrl/data/sram_ctrl_base_testplan.hjson:176]: ahh good catch, thanks!

**Comment by sriyerg** [comment]
LGTM, will defer to @msfschaffner for approval.

**Review by msfschaffner — APPROVED** [review]
LGTM, thanks

**Pull request merged**